### PR TITLE
feat: implement TilePicker component on dashboard module

### PR DESCRIPTION
**Pull request by yvmunayev**
fix: #101 

## Changes proposed in this PR:
- feat: implement TilePicker component on dashboard module
- docs: add doc to TilePicker component 
- test: add unit to Tile component
- test: add unit to TilePicker component

### CODE REVIEW

**Comment by yvmunayev** [comment]
@TahimiLeonBravo @LeandroTorresSicilia @maxxgreene I have doubts with the API specifically with error, label and required

File: packages/dashboard/src/components/Tile/index.js [hunk 1]
```diff
@@ -22,26 +29,32 @@ export default function Tile(props) {
...
+    const context = useContext(TilePickerContext);
     const { ariaDescribedby, groupName, onChange, value, multiple } = context || {};
 
     const inputId = useUniqueIdentifier('tile-input');
 
+    const mainBackground = useTheme().rainbow.palette.background.main;
+    const backgroundColor = isValidColor(backgroundColorProp)
+        ? backgroundColorProp
+        : mainBackground;
+    const color = isValidColor(colorProp) ? colorProp : getContrastText(backgroundColor);
```
Comment (maxxgreene) [inline packages/dashboard/src/components/Tile/index.js:48]: what  you can do is abstract this logic ^^ to a hook `useBackgroundColor()`,

Comment (maxxgreene) [reply packages/dashboard/src/components/Tile/index.js:48]: same with color

**Comment by TahimiLeonBravo** [comment]
the checkmark default color should be "brand"

**Comment by TahimiLeonBravo** [comment]
@yvmunayev the border of the Tile when is selected or hover should have the color transparent gray

File: packages/dashboard/src/components/TilePicker/index.js [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+            <RenderIf isTrue={!!label}>
+                <StyledLabel>
+                    <RequiredAsterisk required={required} />
+                    {label}
+                </StyledLabel>
+            </RenderIf>
+            <StyledOptionsContainer>
+                <Provider value={context}>{children}</Provider>
+            </StyledOptionsContainer>
+            <RenderIf isTrue={!!error}>
```
Comment (maxxgreene) [inline packages/dashboard/src/components/TilePicker/index.js:68]: this logic `!!error` should be done inside of RenderIf, this way we don't have to double negate everytime which is an ugly thing to do :)

**Pull request merged**